docs/user/guides/setup-single-sign-on-google-email.rst

```diff
@@ -0,0 +1,65 @@
+How to setup Single Sign-On (SSO) with Google Workspace
+=======================================================
+
+.. include:: /shared/admonition-rtd-business.rst
+
+This how-to guide will provide instructions on how to enable :abbr:`SSO (Single Sign-on)` with Google Workspace.
+If you want more information on this feature,
+please read :doc:`/commercial/single-sign-on`
+
+Prerequisites
+-------------
+
+.. include:: /_includes/organization-permissions.rst
+
+User setup
+~~~~~~~~~~
+
+Using this setup,
+all users who have access to the configured Google Workspace will be granted a subset of permissions on your organization automatically on account creation.
+
+You can still add outside collaborators and manage their access.
+There are two ways to manage this access:
+
+* Using :doc:`teams </guides/manage-read-the-docs-teams>` to provide access for ongoing contribution.
+* Using :doc:`sharing </commercial/sharing>` to provide short-term access requiring a login.
+
+Enabling SSO
+------------
+
+By default, users that sign up with a Google account do not have any permissions over any project.
+However, you can define which teams users matching your company's domain email address will auto-join when they sign up.
+
+You can enable this feature in your organization:
+
+* Navigate to :guilabel:`<Username dropdown>` > :guilabel:`Organizations` > :guilabel:`<Organization name>` > > :guilabel:`Settings` > :guilabel:`Authorization`
```

Comment:
```suggestion
* Navigate to :menuselection:`"username dropdown" ⬇️ > Organizations > "Organization Name" > Settings > Authorization`
```

Writing this is actually difficult!

Reply:
Looks easier than before? I don't think we need the quotes though?